From 0ecc8b3ce1c56d59565a8e912d2b6d90b660799d Mon Sep 17 00:00:00 2001
From: repo sync <hejiean@eswincomputing.com>
Date: Wed, 2 Jul 2025 13:51:36 +0800
Subject: [PATCH 596/652] chore: move es_proc.h to commom dir

Changelogs:
1. move es_proc.h to include/linux/
2. do not copy header to outside dir

Signed-off-by: hejiean <hejiean@eswincomputing.com>
---
 drivers/memory/eswin/es_proc/Makefile                         | 4 ----
 drivers/memory/eswin/es_proc/es_proc.c                        | 2 +-
 .../memory/eswin/es_proc/include => include}/linux/es_proc.h  | 0
 3 files changed, 1 insertion(+), 5 deletions(-)
 rename {drivers/memory/eswin/es_proc/include => include}/linux/es_proc.h (100%)

diff --git a/drivers/memory/eswin/es_proc/Makefile b/drivers/memory/eswin/es_proc/Makefile
index 9dc204e89fb8e..09e2bc7767a37 100644
--- a/drivers/memory/eswin/es_proc/Makefile
+++ b/drivers/memory/eswin/es_proc/Makefile
@@ -1,7 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0
 obj-$(CONFIG_ESWIN_PROC)	+= es_proc.o
 ccflags-y := -DDEBUG
-
-ES_PROC_HEADER := $(srctree)/drivers/memory/eswin/es_proc/include/linux
-
-COPY_HEADERS := $(shell mkdir -p include/linux && cp $(ES_PROC_HEADER)/*.h include/linux)
diff --git a/drivers/memory/eswin/es_proc/es_proc.c b/drivers/memory/eswin/es_proc/es_proc.c
index 569b9cf307d88..0e955525f0821 100644
--- a/drivers/memory/eswin/es_proc/es_proc.c
+++ b/drivers/memory/eswin/es_proc/es_proc.c
@@ -16,7 +16,7 @@
 #include <linux/proc_fs.h>
 #include <linux/slab.h>
 #include <asm/uaccess.h>
-#include "include/linux/es_proc.h"
+#include <linux/es_proc.h>
 
 static LIST_HEAD(list);
 static es_proc_entry_t *proc_entry = NULL;
diff --git a/drivers/memory/eswin/es_proc/include/linux/es_proc.h b/include/linux/es_proc.h
similarity index 100%
rename from drivers/memory/eswin/es_proc/include/linux/es_proc.h
rename to include/linux/es_proc.h
-- 
2.53.0

